how-to/how-to-add-telerik-ui-for-nativescript-package.md

```diff
@@ -0,0 +1,69 @@
+---
+title: How to Include the {{site.tuin}} Package
+description: A tutorial for adding {{site.tuin}} to a mobile application in AppBuilder. You can use the UI controls by including the free limited edition or by downloading the pro version package from your account and importing it in your app
+type: how-to
+page_title: How to Add {{site.tuin}} in AppBuilder
+slug: include-nativescript-ui-package
+position:
+tags: appbuilder, nativescript
+teampulseid:
+ticketid:
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Package management</td>
+  </tr>
+  <tr>
+    <td>Mobile Development type</td>
+    <td>Native (NativeScript)</td>
+  </tr>
+</table>
+
+## Description
+[{{site.tuin}}](http://www.telerik.com/nativescript-ui) allows you to incorporate native UI controls in your NativeScript applications. You can choose between a free limited edition or a complete edition available with specific subscription plan. To have access to the UI components, you need to add the {{site.tuin}} package to your app in AppBuilder.
```

Comment:
allows you to -> lets you

There's a subtle distinction between the two and I, personally, am inclined to using "lets you/enables you" instead of "allow". Allow has the nuance of "permit" as if there is some restriction, physical or otherwise, while let/enable are more positive and have the nuance of "empower".